src/index.tsx

```diff
@@ -1,13 +1,6 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
-
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Root } from './Root';
 
-import { App } from './App';
-
-createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
-);
+createRoot(document.getElementById('root') as HTMLDivElement).render(<Root />);
```

Comment:
The 'createRoot' method is correctly used to render the 'Root' component into the DOM element with the id 'root'. Ensure that the 'root' element exists in your HTML file, which it does according to the provided index.html file.